datacube/scripts/search_tool.py

```diff
@@ -27,7 +28,7 @@ def printable_values(d):
     return {k: printable(v) for k, v in d.items()}
 
 
-def write_pretty(out_f, field_names, search_results, terminal_size=click.get_terminal_size()):
```

Comment:
Oh my, I don't even think this `datacube-search` thing should exist any more. I don't know why it ever did. The one or two commands it supports should be moved under `datacube dataset` or datacube product`.